### PR TITLE
plplot: fix build with CMake 3.6

### PR DESCRIPTION
**Pull request by tschoonj**
Patch PR is in Homebrew/formula-patches#56

I will update the url to the patch as soon as it is accepted.

### CODE REVIEW

File: Formula/plplot.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@ class Plplot < Formula
     sha256 "b779762659e485d6c9cad54206b1e72f2db5e82950b19a356439e9ce3ef79138" => :mountain_lion
   end
 
+  patch do
```
Comment (zmwangx) [inline Formula/plplot.rb]: Need a comment with links to upstream patches (the SF commits should do) and brief explanation of what the patch does.

**Review by zmwangx — CHANGES_REQUESTED** [review]

**Comment by DomT4** [comment]
Patch was merged into patches as well, so can tweak the link safely.

File: Formula/plplot.rb (hunk 1 above)
Comment (tschoonj) [inline Formula/plplot.rb]: Done!

**Comment by tschoonj** [comment]
As soon as this one is merged in, the formula should build properly again. 

I have asked upstream to make a new release with these patches but they seem to want to hold out until they have finalized some new features they want to include as well.

**Comment by alexant123** [comment]
How I can applying this comment manual and rebuild make instrument?

понедельник, 26 сентября 2016 г. пользователь Tom Schoonjans написал:

**Comment by tschoonj** [comment]
@ilovezfs @zmwangx Could you merge this one in please? Thanks!

**Comment by ilovezfs** [comment]
The job is gone so this will have to be re-queued.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/566054cbee38362cf074cc10fffe7b12c7a38c6a. Thank you for another contribution to Homebrew Tom! 🙇

**Pull request closed without merging**